### DISCUSSION TITLE
I don't understand the magick enumeration system

### DISCUSSION THREAD

**Discussion by LorenzoZappa** [ImageMagick/ImageMagick#7713] [Help] [answered]
### ImageMagick version

7.1.1-38

### Operating system, version and so on

macOS 13.7 (22H123)

### Description

Hi there :)
I am a noob to ImageMagick, but I have a 100ish jpg photos that I wish to convert to bmp format. The simple command I am using is:
magick ./JPG/*.jpg ./BMP/a.bmp
As an output I would like the photos enumerated as a-0.bmp ... a-100.bmp in the same order as the input photos.
But here is the catch I cannot solve.
Every 11th photo is out of order, for example:
photos 24th to 33th are in order, then 34 is out of order, then 35th to 44th are in order, 45 is not, and so on...
If I remove every 11th photo then the resulting one are in the correct order. For example, the photo 35 is the correct followup to photo 33.
I don't know why and what I am doing wrong.
In addition I have noted that this only happens with large batches of files, if I have 20ish or less files this problem doesn't show up.

**Comment by fmw42** [comment, marked as answer]
Images are processed in alphabetic not numeric order.  What are your input file names?

**Reply by LorenzoZappa** [reply]
I am so embarrassed it was so obvious. I have been dealing with this issue for hours and hours. Thank you very much for the help!
